README.md

```diff
@@ -40,6 +40,40 @@ b. To use the icons, click on your preferred icons, and paste inside your body t
         <i class="jini-people1"></i>
     </body>
 
+## How to Start Contributing <a name = "how_contribute"></a>
+1. Fork this repository by clicking on the fork button on the top of this page. This will create a copy of this repository in your account.
+2. Now clone the forked repository to your machine. Go to your GitHub account, open the forked repository, click on the code button and then click the copy to clipboard icon.
+3. Open a terminal and run the following git command:
+      ```
+      git clone "url you just copied"
+      ```
+   where "url you just copied" (without the quotation marks) is the url to this repository (your fork of this project). See the previous steps to obtain the url.
+   For example:
+      ```
+      git clone https://github.com/your_username/bookingjini-icons.git
+      ```
+4. Now create a branch using the 'git switch' command in repository directory:
+      ```
+      git switch -c your-new-branch-name
+      ```
+5. Make necessary changes and commit those changes. Add those changes to the branch you just created using the 'git add' command:
+      ```
+      git add .
+      ```
+    Now commit those changes using the 'git commit' command:
+      ```
+      git commit -m "Add your title here"
+      ```
+6. Push your changes using the command git push:
+      ```
+      git push -u origin your-branch-name
+      ```
+      replacing your-branch-name with the name of the branch you created earlier.
+      
+7. Submit your changes for review 
+    If you go to your repository on GitHub, you'll see a Compare & pull request button. Click on that button. Now add appropriate title and comment on it and submit
+    the pull request.
+
 ## Contributors
```

Comment:
please don't add this part now as this is the first time we are participating in open source

Reply:
@AshutoshDash1999 Sure I'll add screenshots of each steps to improve it, but before let me know which part I should not add can you clarify more?

Reply:
The contributors part

Reply:
Okay I didn't added that It was already there before, well If u think it's not necessary I'll remove it and I'll add screenshots and will send PR ASAP.. thank you for clarify.

Reply:
hey @Jiggy9, sorry my bad, I thought the contributors part that you are adding is blank. So I meant not to set it blank. You can go forward with it. Just add screenshots of every step. That will be great.